testing/functional_tests/test_automate_training.py

```diff
@@ -10,7 +11,7 @@ def setup_function():
 
 
 @pytest.mark.script_launch_mode('subprocess')
-def test_automate_training(script_runner):
+def test_automate_training(get_functional_test_files, script_runner):
```

Comment:
`def test_automate_training_run_test(script_runner):` needs the fixture `get_functional_test_files`

Reply:
Good catch. Without it, that specific unit test would have fail and my folder/module level pytest check would never have never picked it up. Good call!  Added.